sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/TransformWithStateExec.scala

```diff
@@ -115,7 +119,7 @@ case class TransformWithStateExec(
    * Fetching the columnFamilySchemas from the StatefulProcessorHandle
    * after init is called.
    */
-  private def getColFamilySchemas(): Map[String, StateStoreColFamilySchema] = {
+  def getColFamilySchemas(): Map[String, StateStoreColFamilySchema] = {
     val columnFamilySchemas = getDriverProcessorHandle().getColumnFamilySchemas
     closeProcessorHandle()
     columnFamilySchemas
```

Comment:
should this be moved to a static method?

Reply:
Good question - I guess if we can pass the stateful processor in, it can be.

Reply:
Actually, I don't think you can make it static - we need the the `statefulProcessor` that is passed into this particular instance of the `TransformWithStateExec` class.